src/sql/workbench/browser/modelComponents/media/table.css

```diff
@@ -25,3 +22,36 @@
 .modelview-table-button-icon {
 	background-color: transparent;
 }
+
+modelview-table .detailView-toggle {
+	display: inline-block;
+	cursor: pointer;
+}
+
+modelview-table .dynamic-cell-detail {
+	z-index: 101;
+	position: absolute;
+	margin: 0;
+	padding: 0;
+	width: 100%;
+	overflow: auto;
+}
+
+modelview-table .dynamic-cell-detail> :first-child {
+	vertical-align: middle;
+	line-height: 13px;
+	padding: 10px;
+	margin-left: 20px;
+}
+
+modelview-table .dynamic-cell-detail>.detail-container {
+	overflow: auto;
+	display: block !important;
+	max-height: 100px !important;
+	line-height: 20px;
+}
+
+modelview-table .dynamic-cell-detail>.detail-container>div,
```

Comment:
Please add spaces between the elements (e.g. `.dynamic-cell-detail > .detail-container > div`) Makes it a lot easier to read